.travis.yml

```diff
@@ -2,16 +2,16 @@ sudo: false
 language: ruby
 
 rvm:
-  - 2.5.0
-  - 2.4.3
-  - 2.3.6
-  - 2.2.9
-  - 2.1.10
+  - "2.6"
+  - "2.5"
+  - "2.4"
+  - "2.3"
+  - "2.2"
 
 cache: bundler
 
 before_install:
-  - gem update --system
```

Comment:
`gem update --system` fails on older rubies, because rubygems dropped the compat for `MRI < 2.3`.